appcli/configuration_manager.py

```diff
@@ -206,6 +251,34 @@ def set_variable(self, variable: str, value: any):
     def get_stack_variable(self, variable: str):
         return self.variables_manager.get_stack_variable(variable)
 
+    def __validate_schema(self, config_file: Path, schema_file: Path) -> None:
+        """Attempt to validate a config file against a provided schema file.
+        The function will try and determine the file content based off the extenstion.
+        It will throw if the extension is unknown or the contents do not match the expected format.
+
+        Args:
+            config_file (Path): Path to the config file to validate.
+            schema_file (Path): Path to the schema file.
+        """
+        # Check we actually have a loader for the filetype.
+        try:
+            loader = FILETYPE_LOADERS[config_file.suffix]
+        except KeyError:
+            error_and_exit(
+                f"The {config_file} file does not have an associated loader function. Check that the suffix is a supported type."
```

Comment:
It's helpful to the user if we tell them what the supported extensions are when we throw a message like this. Otherwise they have to go look it up / work it out onsite. Just something simple like the following is fine:
```
f"..... Check that the suffix is one of the supported types: {[k for k in FILETYPE_LOADERS.keys()]}"
```

Reply:
Also, in our messages, let's wrap the file name in backticks for clarity.
```
f"The `{config_file}` file....
```